Add documentation on how to use Weblate for translations (#1687)
plone/documentation: docs/i18n-l10n/contributing-translations.md

4. Commit your changes, and create a pull request.

> (contributing-plone-core-translations-using-weblate)=

Reviewer: ```suggestion
(contributing-weblate-for-translations)=
```

> ## Using Weblate to translate Plone

Reviewer: ```suggestion
## Weblate for translations
```

> [Weblate](https://weblate.org/) is an Open Source project to help software developers translate their projects.

Reviewer: ```suggestion
[Weblate](https://weblate.org/) is an open source project to help software developers translate their projects.
```

> It lets the translator to work in a web based interface, without the need of installing any third-party software or without the need to use git or GitHub.

Reviewer: ```suggestion
Translators can work in a web interface, and not have to install third-party software or use git or GitHub.
```

> Plone Foundation has obtained a so called `libre` account for Plone, meaning we get the hosting with many other Open Source Projects at the [Hosted](https://hosted.weblate.org/) platform that Weblate offers to Open Source projects.

Reviewer: ```suggestion
The Plone Foundation has obtained a "gratis Libre plan" account for Plone.
Plone gets free hosting at the [Hosted](https://hosted.weblate.org/) platform that Weblate offers to open source projects.
```

> This means that the translator can go to the [Plone project and Weblate](https://hosted.weblate.org/projects/plone/) and start translating there.

Reviewer: ```suggestion
Translators can go to the [Plone project in Weblate](https://hosted.weblate.org/projects/plone/) to write translations.
```

> You will need to create an account there and then your translations will be automatically contributed back to Plone's GitHub repository.

Reviewer: ```suggestion
Translators will need to create an account there.
Then your translations will be automatically contributed back to Plone's GitHub repository.
```

```{note}

> At this moment Volto translations can't be contributed using Weblate. We are working on this issue and we hope to have it fixed as soon as possible.

Reviewer: ```suggestion
At this moment Volto translations can't be contributed using Weblate.
The Volto Team are working on this issue, and hope to fix it as soon as possible.
```
